Makefile

```diff
@@ -189,11 +189,16 @@ verify_boilerplate:
 verify_codegen:
 	$(HACK_DIR)/verify-codegen.sh
 
+verify_vendor: $(BINDIR)/dep
+	dep ensure -no-vendor -dry-run -v
+
+verify_gen_docs: verify_reference_docs verify_cmd_docs
+
 verify_reference_docs: $(BINDIR)/node
 	$(HACK_DIR)/verify-reference-docs.sh
 
-verify_vendor: $(BINDIR)/dep
-	dep ensure -no-vendor -dry-run -v
+verify_cmd_docs:
```

Comment:
Shouldn't we also add a `go_cmd_docs_gen` command?

Reply:
Not sure what you mean

Reply:
I see what you mean now, yes that makes sense